cartographer_ros/src/cartographer_node_main.cc

```diff
@@ -634,6 +662,82 @@ void Node::PublishPose(const int64 timestamp) {
   last_pose_publish_timestamp_ = timestamp;
 }
 
+void Node::PublishOccupancyGrids() {
```

Comment:
PublishOccupanyGrid, there's only the one grid. I think using the plural is the wrong way to indicate that this will loop forever. Maybe ContinuouslyPublishOccupanyGrid? Or PublishOccupancyGridForever?